beluga/routes.py

```diff
@@ -4,8 +4,12 @@
 from sanic.exceptions import abort
 from sanic.response import json
 
+from beluga.models import session_scope, Event
 from beluga.auth import authorized
 
+import json as pyjson
+from datetime import datetime as dt, timezone as tz
+import math
```

Comment:
[Nitpick]
Idiomatic import ordering: base packages, 3rd party packages, then local modules, all in alphabetical order. 
☝️ These 3 are base packages.